openshift/jobs.yml

```diff
@@ -0,0 +1,165 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: ${APP_NAME}-cronjob
+parameters:
+  - name: APP_NAME
+    description: App name that OpenShift objects will be prefixed with
+    required: true
+  - name: IMAGE_NAMESPACE
+    description: Namespace from which to pull the image
+    required: true
+  - name: IMAGE_TAG
+    description: Image tag from image stream
+    value: latest
+    required: true
+  - name: CONFIG_VERSION
+    description: The github repository version of Config (Commit SHA)
+    value: default
+spec:
+    schedule: '*/5 * * * *'
```

Comment:
I'd err on the side of caution for the committed / deployable script here. Ie. once a month or something. With a note to change this depending on environment